addons/sale_product_configurator/controllers/main.py

```diff
@@ -56,14 +56,9 @@ def _optional_product_items(self, product_id, pricelist, **kw):
             'add_qty': add_qty,
         })
 
-    def _show_optional_products(self, product_id, variant_values, pricelist, handle_stock, **kw):
+    def _show_advanced_configurator(self, product_id, variant_values, pricelist, handle_stock, **kw):
```

Comment:
I don't know why every controller here do:

```python
@http.route(/xxx)
def my_method(a,b,c):
    return self._my_method(...)

def _my_method(...)
    # do stuff here
```

When `_my_method()` is only called from that controller.. what's the point of that encapsulation :shrug: 

But it was like that before so.. keep it like that and ignore this message :grimacing: